packages/graphql/src/queries/AllFlights.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import { GraphQLNonNull } from 'graphql';
+import {
+  connectionArgs,
+  connectionDefinitions,
+  connectionFromArray,
+  type ConnectionArguments,
+} from 'graphql-relay';
+
+import FlightInput from '../types/input/FlightSearchInput';
+import GraphQLFlight from '../types/output/Flight';
+import type { GraphqlContextType } from '../services/GraphQLContext';
+import type { Flights } from '../dataloaders/Flightsloader';
+
+const { connectionType: AllFlightsConnection } = connectionDefinitions({
+  nodeType: GraphQLFlight,
+});
+
+type Args = {|
+  +input: {|
+    +flyFrom: string,
+    +dateFrom: Date,
+    +dateTo?: Date,
+  |},
+  ...$Exact<ConnectionArguments>,
+|};
+
+export default {
+  name: 'AllFlights',
```

Comment:
We should not forget we are not about flights anymore. This query can return also trains and buses, in the future also ferries etc.

Based on [our gossary](https://kiwi.wiki/kiwi/handbook/#/comms/glossary?id=itinerary-parts-examples) this query returns itineraries so shall we named it `itineraries` or `searchItineraries` maybe?

Reply:
I think itineraries is fine, I will change it